hw/ip/entropy_src/dv/env/entropy_src_env.core

```diff
@@ -8,7 +8,7 @@ filesets:
   files_dv:
     depend:
       - lowrisc:dv:ralgen
-      - lowrisc:dv:rng_agent
+      - lowrisc:dv:push_pull_agent
```

Comment:
do we still need rng_agent in the repo, or it can be removed?

Reply:
I was going to leave rng_agent in the repo until I got further along in dv just to make sure I can do everything with the push_pull_agent before deleting it.

Reply:
sounds good